docs/user_guide/assessment.rst

```diff
@@ -42,17 +130,26 @@ of the ten cases where the true value is `1`, so we expect the recall to be 0.5:
 
 .. _metrics_with_grouping:
 
-Metrics with Grouping
-^^^^^^^^^^^^^^^^^^^^^
+Metrics with grouping using :code:`MetricFrame`
+-----------------------------------------------
 
-When considering fairness, each row of input data will have an associated
+In a typical fairness assessment, each row of input data will have an associated
 group label :math:`g \in G`, and we will want to know how the metric behaves
-for each :math:`g`. To help with this, Fairlearn provides a class, which takes
-an existing (ungrouped) metric function, and applies it to each group within a
+for each :math:`g`. To help with this, Fairlearn provides a class that takes
+an existing (ungrouped) metric function and applies it to each group within a
 set of data.
 
-Suppose in addition to the :math:`Y_{true}` and :math:`Y_{pred}` above, we had
-the following set of labels:
+This data structure, :class:`fairlearn.metrics.MetricFrame`, enables evaluation 
+of disaggregated metrics. In its simplest form :class:`fairlearn.metrics.MetricFrame` 
+takes four arguments:
+* metric_function with signature metric_function(y_true, y_pred)
+* y_true: array of labels
+* y_pred: array of predictions
+* sensitive_features: array of sensitive feature values
```

Comment:
Is this intended to be rendered as a list?

Reply:
Yes, I fixed the spacing issues. Thanks!